### PR TITLE
Integrations: update GitHub guides

### PR DESCRIPTION
**Pull request by stevenrombauts**
This PR updates the GitHub guides for Cloud and Self-Hosted to bring them up to date with the current UI and requirements.

### CODE REVIEW

File: docs/self-hosted/github-integration-setup.md [hunk 1]
```diff
@@ -1,28 +1,50 @@
 # GitHub (Enterprise) Integration Setup
 ## Private Packagist Self-Hosted
 
-This guide explains how to setup an OAuth integration for Private Packagist Self-Hosted with either the on-premises version of GitHub, or their public service on github.com.  
-If you are using our cloud product at packagist.com, [use this guide](../cloud/github-integration-setup.md).
+<div class="row column">
+    <div class="callout success">
+        <p>This guide explains how to setup an OAuth integration for Private Packagist Self-Hosted with either the on-premises version of GitHub, or their public service on GitHub.com.</p>
```
Comment (glaubinix) [inline docs/self-hosted/github-integration-setup.md:6]:
```suggestion
        <p>This guide explains how to set up an OAuth integration for Private Packagist Self-Hosted with either the on-premises version of GitHub, or their public service on GitHub.com.</p>
```

File: docs/cloud/github-integration-setup.md [hunk 2]
```diff
@@ -1,31 +1,55 @@
-# GitHub (Enterprise) Integration Setup
+# GitHub Enterprise Integration Setup
 ## Private Packagist Cloud
 
-## Initial Setup
+<div class="row column">
+    <div class="callout success">
+        <p>
+            This guide explains how to setup an OAuth integration for Private Packagist Cloud with GitHub Enterprise Cloud/Server.
```
Comment (glaubinix) [inline docs/cloud/github-integration-setup.md:7]:
```suggestion
            This guide explains how to set up an OAuth integration for Private Packagist Cloud with GitHub Enterprise Cloud/Server.
```

File: docs/cloud/github-integration-setup.md [hunk 3]
```diff
@@ -1,31 +1,55 @@
...
+
+To allow your users to authenticate to Private Packagist with their GitHub Enterprise account, you'll first need to create 
+an integration in Private Packagist. This document walks you through the required steps. 
+
+## Open the Add Integration form
 
 From the organization settings page, select the "Integrations" subtab. Hit the "Add Integration" button on the integrations listing page to get to the form below.
 To set up a GitHub Enterprise integration with Private Packagist start by selecting "GitHub" as the platform and enter the URL of your GitHub Enterprise server, as seen in the example below.
-A link to setup the OAuth application on your GitHub enterprise server will automatically be displayed.
+A link to create a new OAuth application on your GitHub enterprise server will automatically be displayed.
```
Comment (glaubinix) [inline docs/cloud/github-integration-setup.md:21]:
```suggestion
A link to create a new OAuth application on your GitHub Enterprise server will automatically be displayed.
```
This is capitalized everywhere else

File: Resources/public/img/docs/integration-setup/self-hosted/github-04-packagist-form.png
Comment (glaubinix) [inline Resources/public/img/docs/integration-setup/self-hosted/github-04-packagist-form.png:1]: Everywhere else you were hiding the client ID but here it is shown

**Review by glaubinix — COMMENTED** [review]
Looks clear and updated screenshots match.

There are two image (github-enterprise-01-..., github-enterprise-03) that you edited in place. I don't fully remember this but I think you might have to change the name to force clear the browser cache

File: docs/self-hosted/github-integration-setup.md [hunk 4]
```diff
@@ -1,28 +1,50 @@
...
+</div>
 
-## Initial Setup
-Hit the “Add Integration“ button on the admin page to get to the form below.
+To allow your users to authenticate to Private Packagist Self-Hosted with their GitHub or GitHub Enterprise account, you'll 
+first need to create an integration in Private Packagist. This document walks you through the required steps.
+
+## Open the Add Integration form
+
+Log in with an admin account, go to the Admin section and hit the “Add Integration“ button to open the form to create your integration.
```
[the remark this answers is not in the document]
Comment (stevenrombauts) [reply docs/self-hosted/github-integration-setup.md:16]: @glaubinix So do you want to mention both cases (setup mode on/off), or just leave out the "login" part?

[the remark this answers is not in the document]
Comment (glaubinix) [reply docs/self-hosted/github-integration-setup.md:16]: For self-hosted the most common case will be, setup mode is on and a devops/admin person configures the application. Would probably mention both.

[the remark this answers is not in the document]
Comment (stevenrombauts) [reply docs/self-hosted/github-integration-setup.md]: Got it, added some more explanation about this to this paragraph then.

[the remark this answers is not in the document]
Comment (glaubinix) [reply docs/self-hosted/github-integration-setup.md]: Thanks! Looks good!

**Review by glaubinix — APPROVED** [review]

**Review by timscharfenort8 — COMMENTED** [review]
Good work

**Pull request merged**